lib/core/client.js

```diff
@@ -114,6 +115,21 @@ class NightwatchAPI {
   isOpera() {
     return this.capabilities.browserName === Browser.OPERA;
   }
+
+  isAppiumClient() {
+    if (this.options.selenium && this.options.selenium.use_appium) {
+      return true;
+    }
+
+    // Handle BrowserStack case
+    // (BrowserStack always returns platformName in capabilities)
```

Comment:
What about other cloud providers like sauce labs, and lambda tests? Does this method work out of the box for them?

Reply:
Since we don't have native support for other cloud providers, we'd expect users to set `use_appium` property when using Appium with other cloud providers. And so, this method will work for them as well. (When testing with BrowserStack, `use_appium` property is optional and users may/may not use it).